Week3/more_javascript.js

```diff
@@ -0,0 +1,183 @@
+/*1- Create a function that takes 3 arguments and returns the sum of the these arguments.*/
+function sum(x, y ,c) {
+   return x+y+c;
+}
+console.log(sum(2,3,5)); 
+
+/* 2- Create a function named colorCar that receives a color, and prints out, 'a red car' for example.*/
+function colorCar(color){
+console.log("a %o car", color)
```

Comment:
ah, neat! Please don't forget to add `;` at the end of the `console.log`. That will help you avoid small mistakes in the future

Reply:
done +1: